File: abc/abc154/a.cpp
// https://atcoder.jp/contests/abc154/submissions/26232175

#include <bits/stdc++.h>
using namespace std;
int main() {
  string s, t, u;
  int a, b;
  cin >> s >> t >> a >> b >> u;
  if(s == u) a--;
  else b--;
  cout << a << endl << b;
}

File: abc/abc154/a.java
// https://atcoder.jp/contests/abc154/submissions/43257428

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    String s = stdIn.next();
    String t = stdIn.next();
    int a = stdIn.nextInt();
    int b = stdIn.nextInt();
    String u = stdIn.next();

    if(s.equals(u)) {
      a--;
    } else {
      b--;
    }

    System.out.printf("%d %d", a, b);
  }
}
